templates/has_joined_cluster.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+# Check if cluster node is ready and has
+
+# Config
+# ------
+total_maxwait_seconds={{ mariadb.galera.join.max_wait }}
+# Pause zwischen 2 Versuchen in Sekunden
```

Comment:
I think we should use English here as well.

Reply:
fixed in 019bae1a81abc91428caa08566e9a30cd8a215a9